must-gather/collection-scripts/gather

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+CMD=/usr/bin/oc
+STYLE=json
+
+show_header() {
```

Comment:
I didn't find any official recommendation about the output format, but kubevirt and performance addon operator are less verbose / "nice". Just curious where you did get this from?

Reply:
looking further... I think writing to stdout is wrong. I understand that everything should be written into a `must-gather` dir. Please see how others use `oc adm inspect --dest-dir must-gather <resource> `

Reply:
here https://github.com/kubevirt/must-gather/blob/master/collection-scripts/gather they all write some text to stdout. Also here https://github.com/kubevirt/must-gather/blob/master/Dockerfile it says that the gather script is the entrypoint of the built image.

Reply:
I thought the text output should have some headers for better readability. Why is that wrong?

Reply:
as discussed in chat:
some debugging can be printed to stdout
but manifests need to be in a `must-gather` dir, preferable with `oc adm inspect --dest-dir must-gather <resource>` like the other must gather images do as well 
otherwise the manifest won't be copied to your local must-gather dir